internal/actions.go

```diff
@@ -28,17 +28,18 @@ func downloadPemKey(app string) {
 	os.Chmod(filepath.Join(downloadDir, "server.pem"), 0600)
 }
 
-func GetAppConfiguration(app string, path string) bool {
+func GetAppConfiguration(app string) bool {
```

Comment:
i know we can name return variables in Go, maybe do so here? it's not immediately clear why a `GetConfiguration` function would return a `bool`, but if it was named `success` it would be much more clear